README.md

```diff
@@ -1 +1,29 @@
-# Operator Marketplace
+# Marketplace Operator
+
+## Project Status: pre-alpha
```

Comment:
Glad that this info is making it into the readme, though I'm not sure it makes sense to be formatted as a header.

Reply:
This follows the standard shown [here](https://github.com/operator-framework/operator-sdk#project-status-pre-alpha)

Reply:
Added the following under the Header in question.
> The project is currently pre-alpha and it is expected that breaking changes to the API will be made in the upcoming releases.

Reply:
Wasn't aware of that example, I think the added line clarifying makes it much clearer :)

Reply:
Great, please feel free to resolve any conversations in this PR that met your expectations.